packages/mobile/src/utils/permissions.android.ts

```diff
@@ -26,7 +28,9 @@ export async function checkContactsPermission() {
 }
 
 async function requestPermission(permission: Permission, title?: string, message?: string) {
+  const { dispatch } = store
```

Comment:
This use of the store outside saga-land potentially problematic, especially for async functions like this.
I don't fully understand the implications of accessing the store this way but I believe it's frowned upon. Let's look into it

Reply:
As far as I know `dispatch` can be used outside saga-land. As an example, we are using it here:
https://github.com/celo-org/celo-monorepo/blob/cf13c179263060a968ef1086bbabe6230680f6ec/packages/mobile/src/app/App.tsx#L58-L60